.github/workflows/tests.yml

```diff
@@ -0,0 +1,43 @@
+name: taskchampion
```

Comment:
Currently the "Checks" tab has two sections titled "taskchampion", would possibly be neater if this said "tests" or similar?

There's also two checks titled "Clippy" which seems slightly odd (not sure if they are doing different things?)

Reply:
I kind of don't understand the Actions UI and where it gets names from.  I'm happy to see this addressed in another PR.